packages/hardhat-core/sample-projects/advanced-ts/hardhat.config.ts

```diff
@@ -0,0 +1,45 @@
+import * as dotenv from "dotenv";
+
+import { HardhatUserConfig, task } from "hardhat/config";
+import "@nomiclabs/hardhat-etherscan";
+import "@nomiclabs/hardhat-waffle";
+import "@typechain/hardhat";
+import "hardhat-gas-reporter";
+import "solidity-coverage";
+
+dotenv.config();
+
+// This is a sample Hardhat task. To learn how to create your own go to
+// https://hardhat.org/guides/create-task.html
+task("accounts", "Prints the list of accounts", async (taskArgs, hre) => {
+  const accounts = await hre.ethers.getSigners();
+
+  for (const account of accounts) {
+    console.log(account.address);
+  }
+});
+
+// You need to export an object to set up your config
+// Go to https://hardhat.org/config/ to learn more
+
+const config: HardhatUserConfig = {
+  solidity: "0.8.4",
+  networks: {
+    hardhat: {
+      initialBaseFeePerGas: 0, // workaround from https://github.com/sc-forks/solidity-coverage/issues/652#issuecomment-896330136 . Remove when that issue is closed.
+    },
+    ropsten: {
+      url: process.env.ROPSTEN_URL || "",
+      accounts: [process.env.PRIVATE_KEY || `0x${"0".repeat(40)}`],
+    },
+  },
+  gasReporter: {
+    enabled: process.env.REPORT_GAS !== undefined,
+    currency: "USD",
```

Comment:
@feuGeneA 

Hi, wrt your issue at [hardhat-gas-reporter 72][1]. Was not able to reproduce (using a valid coinmarketcap api. key). I know you tried that but ... maybe there was a copy-paste error or something?  

Price data seems to be working ok for latest hardhat in [this recent CI run for the plugin][2].

The config used for that test is:
```js
gasReporter: {
    coinmarketcap: process.env.CMC_API_KEY
 }
```



[1]: https://github.com/cgewecke/hardhat-gas-reporter/issues/72
[2]: https://app.circleci.com/pipelines/github/cgewecke/hardhat-gas-reporter/58/workflows/1236c142-0ef3-4563-80fb-f6685a7a7053/jobs/57

Reply:
@cgewecke, is there a way to disable this feature and just show the prices in eth? While i find it really cool, it would be nice if we don't ask the users to provide another api key in these sample projects.

Reply:
Yes good idea,. Someone else has requested that here as well ... (cross-linking: https://github.com/cgewecke/eth-gas-reporter/issues/219)